integration-libs/opf/checkout/components/b2b/opf-b2b-checkout-payment-type/opf-b2b-checkout-payment-type.component.html

```diff
@@ -0,0 +1,56 @@
+<h2 class="cx-checkout-title d-none d-lg-block d-xl-block">
+  {{ 'checkoutB2B.methodOfPayment.paymentType' | cxTranslate }}
+</h2>
+<ng-container
+  *ngIf="(typeSelected$ | async) && !(isUpdating$ | async); else loading"
+>
+  <ng-container>
+    <div
+      *cxFeature="'!a11yRemoveStatusLoadedRole'"
+      role="status"
+      [attr.aria-label]="'common.loaded' | cxTranslate"
+    ></div>
+    <div class="row">
+      <div class="col-md-12 col-lg-6">
+        <label>
+          <span class="label-content">{{
+            'checkoutB2B.poNumber' | cxTranslate
+          }}</span>
+          <input
+            #poNumber
+            class="form-control"
+            formControlName="poNumber"
+            type="text"
+            placeholder="{{ 'checkoutB2B.placeholder' | cxTranslate }}"
+            value="{{ cartPoNumber$ | async }}"
```

Comment:
does it affect perf to has a sync directly in form attribute?
wondering if we should subscribe it once in 'div' or 'container'?

Reply:
I think it may affect performance. I will try to fix this.